packages/web/src/components/ui/Card/Card.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import styles from './Card.module.scss';
+import clsx from 'clsx';
+import { numberWithSpaces } from '../../../utils/helpers';
+import { Bug } from 'components/icons';
+import Chip from '../Chip/Chip';
+
+export type CardProps = {
+  title: string;
+  icon?: string;
+  description?: string;
+  packageTags?: {
+    featuredPackages: string[];
+    restPackages: number;
+  };
+  featuredSites?: {
+    iconList: string[];
+    numberOfUses: number;
+  };
+  vulnerablePackage?: {
+    name: string;
+    moreCount?: number;
+  };
+  variant?: 'to-all' | 'vulnerable' | string;
+};
+
+export default function Card({
+  title,
+  icon,
+  description,
+  packageTags,
+  featuredSites,
+  vulnerablePackage,
+  variant = '',
+}: CardProps) {
+  return (
+    <div className={clsx(styles.card, styles[variant])}>
+      <div className={styles.cardTop}>
+        <header className={styles.header}>
+          {icon && (
+            <div className={styles.icon}>
+              <img src={icon} alt='' />
+            </div>
+          )}
+
+          <div className={styles.title}>{title}</div>
+        </header>
+
+        {description && <div className={styles.description}>{description}</div>}
+      </div>
+
+      {variant === 'toAll' && (
```

Comment:
Is it `to-all` or `toAll`? Also could you please remove `string` type from `variant` field to avoid such mistakes.